### ISSUE TITLE
Show bundled file size in readme

### ISSUE THREAD

**Issue by stevenvachon** [matthewp/haunted#95]
It's important, I think, if the goal is to communicate that this can replace React in more ways than function.

**Comment by matthewp** [comment]
Yep, good point. I think there's some badges out there that automate this sort of thing too.

**Comment by stevenvachon** [comment]
My experience with those automated badges is that they show the package size and not the bundled size. Because of this, I manually update a custom badge's text (via a URL within the readme) for my personal projects.

**Comment by matthewp** [comment]
@stevenvachon Thanks! Do you have an example where you do this?

**Comment by stevenvachon** [comment]
@matthewp the readme for https://github.com/stevenvachon/scrolling-menu
